sunpy/coordinates/frameattributes.py

```diff
@@ -127,11 +127,8 @@ def _convert_string_to_coord(self, out, obstime):
     def __get__(self, instance, frame_cls=None):
         # If instance is None then we can't get obstime so it doesn't matter.
         if instance is not None:
-            # Get observer if the instance has one, or the default.
-            observer = getattr(instance, '_' + self.name, self.default)
-
-            # We have an instance of a frame, so get obstime
-            obstime = getattr(instance, 'obstime', None)
+            observer = getattr(instance, '_' + self.name)
+            obstime = getattr(instance, 'obstime', None)  # TODO: Why is this `None` needed?
```

Comment:
I'm not sure why this line is getting called on objects without `obstime`, but there can be errors during transformations of the `None` fallback is not present.  Curiously, it's not needed for the `observer` line just above it.   I've marked it with a comment for future investigation (beyond the scope of this PR).